src/common.ts

```diff
@@ -109,25 +149,25 @@ export const zomeCallSpec = ({zomeName, funcName}) => (
   `${zomeName}/${funcName}`
 )
 
-/**
- * Make a zome call through the WS client, identified by AgentID + DNA Hash
- */
-export const zomeCallByDna = async (client, {agentId, dnaHash, zomeName, funcName, params}) => {
-  let instance = await lookupHoloInstance(client, {dnaHash, agentId})
-  const instanceId = instanceIdFromAgentAndDna(instance.agentId, instance.dnaHash)
-  return zomeCallByInstance(client, {instanceId, zomeName, funcName, params})
+
+type CallFnParams = {
+  instanceId: string,
+  zomeName: string,
+  funcName: string,
+  args: any
 }
 
 /**
  * Make a zome call through the WS client, identified by instance ID
  * TODO: maybe keep the Ok/Err wrapping, to differentiate between zome error and true exception
  */
-export const zomeCallByInstance = async (client, {instanceId, zomeName, funcName, params}) => {
+export const zomeCallByInstance = async (client, callParams: CallFnParams) => {
+  const {instanceId, zomeName, funcName, args} = callParams
```

Comment:
```suggestion
  const {instanceId: instance_id, zomeName: zome, funcName, args} = callParams
```
and then:
```
  const payload = {
    instance_id,
    zome,
    function: funcName,
    args,
  }
```
What do you think about this code style? Is this more readable, or is it confusing with TS syntax?

Reply:
Not confusing, but I'm wondering what the need is for changing instanceId to instance_id.  Shouldn't we try to consistently use camelcase?